app/controllers/github_events_controller.rb

```diff
@@ -33,8 +33,8 @@ def process_pull_request_comment
       review.reject if comment.rejected?
       review.approve if comment.approved?
     elsif comment.commenter == owner
-      comment.parsed_body.mentions.each do |username|
-        pull_request.reviews.create(github_user: username)
+      comment.parsed_body.mentions.each do |github_user|
+        pull_request.reviews.create_or_reset_for(github_user)
```

Comment:
Ha, I thought you were just proposing names :-) Didn't know they were an actual thing.